packages/p2p/src/components/file-dropzone/fade-in-message/fade-in-message.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { CSSTransition } from 'react-transition-group';
+import { Text } from '@deriv/components';
+
+type TFadeInMessage = {
+    is_visible: boolean;
+    color?: string;
+    key?: string;
+    timeout: number;
+    no_text?: boolean;
+};
+
+const FadeInMessage = ({
+    children,
+    color,
```

Comment:
can we have default values for optional params?

Reply:
hmmmm in this case, I dont think we need to since in the component itself, it already handles if the prop was left undefined